k8s/konghq.tf

```diff
@@ -7,6 +7,11 @@ resource "helm_release" "konghq-external" {
   namespace  = kubernetes_namespace_v1.kong.metadata[0].name
   version    = "2.12.0"
 
+  set {
+    name = "affinity"
+    value = templatefile("${path.module}/configs/konghq/antiaffinity.yaml", {app = helm_release.konghq-internal.name})
```

Comment:
I'm not sure you need to use helm_release.konghq-internal.name for the external kong resource

Reply:
It's just that the affinity rule needs to know which pod to avoid